From e8642d318e96e00899381cfff47ee4e0354aa0a1 Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Thu, 10 Mar 2022 14:30:01 +0800
Subject: [PATCH 2/3] cmake: exclude Yocto build flags

Ipp-crypto has its own set of flags and -march values,
which causes conflict with default -march=nehalem in gcc

Upstream-Status: Inappropriate

Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
 sources/ippcp/crypto_mb/src/cmake/linux/GNU.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sources/ippcp/crypto_mb/src/cmake/linux/GNU.cmake b/sources/ippcp/crypto_mb/src/cmake/linux/GNU.cmake
index 9656a77d..3c54e6d5 100644
--- a/sources/ippcp/crypto_mb/src/cmake/linux/GNU.cmake
+++ b/sources/ippcp/crypto_mb/src/cmake/linux/GNU.cmake
@@ -54,7 +54,7 @@ endif()
 # Compiler flags
 
 # Tells the compiler to align functions and loops
-set(CMAKE_C_FLAGS " -falign-functions=32 -falign-loops=32")
+set(CMAKE_C_FLAGS " -falign-functions=32 -falign-loops=32 ${TOOLCHAIN_OPTIONS}")
 # Ensures that compilation takes place in a freestanding environment
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -ffreestanding")
 
-- 
2.37.3

